src/renderer/components/+workloads/workloads.tsx

```diff
@@ -115,7 +116,7 @@ export class Workloads extends React.Component {
 
   render() {
     return (
-      <TabLayout className="Workloads" tabs={Workloads.tabRoutes}/>
+      <TabLayout className="Workloads" tabs={Workloads.tabRoutes} />
```

Comment:
Are you using some formatter for these whitespace fixes? It adds a bit noise to the PR.

Reply:
I think I had vscode's default formatter turned on. I have turned it off. I'll revert these changes.